supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -720,7 +720,7 @@ Therefore, if you need to use the indefinite article before "sosreport", use _an
 // OCP: Added "In Red Hat OpenShift,"
 [[spec]]
 ==== image:images/yes.png[yes] spec (noun)
-*Description*: In Red Hat OpenShift, in addition to "spec file", which is permitted when it relates to RPM spec files, you can also use "spec" for general usage when you describe Kubernetes or OpenShift Container Platform object specs, manifests, or definitions.
+*Description*: In Red Hat OpenShift, in addition to "spec file", which is permitted when it relates to an RPM's spec file, you can also use "spec" for general usage when you describe Kubernetes or OpenShift Container Platform object specs, manifests, or definitions.
```

Comment:
I don't think using "RPM's spec file" sounds right. I think it would still just be an "RPM spec file"

Reply:
Thanks. Updated.